cloud/amazon/ec2_vpc_route_table.py

```diff
@@ -19,18 +19,21 @@
 short_description: Manage route tables for AWS virtual private clouds
 description:
     - Manage route tables for AWS virtual private clouds
-version_added: "2.0"
-author: Robert Estelle (@erydo), Rob White (@wimnat)
+version_added: "2.2"
```

Comment:
I don't think this should be changed, since this isn't new in the 2.2/2.3 release.